### DISCUSSION TITLE
Em quais versões do delphi funciona ?

### DISCUSSION THREAD

**Discussion by fcaucotto** [viniciussanchez/RESTRequest4Delphi#155] [General]
Ola, em qual versões do delphi funciona ?
Estou utilizando o delphi XE5.

Obrigado

**Comment by viniciussanchez** [comment]
Nunca testei em versões menores que 10.3
mas deve funcionar, qualquer coisa pode ate usar outros motores, sem ser o padrão

**Reply by fcaucotto** [reply]
Valeu, obrigado.
